dGame/Character.cpp

```diff
@@ -15,6 +15,7 @@
 #include "Zone.h"
 #include "ChatPackets.h"
 #include "Inventory.h"
+#include "InventoryComponent.h"
```

Comment:
why are there a bunch of header additions of InventoryComponent and CDClientManager?

Reply:
Because I removed the InventoryComponent include in GameMessages.h to reduce the number of pre-compiled headers.  This caused several dozen files to no longer compile since they were no longer included anymore.